corgi/collectors/errata_tool.py

```
@@ -75,40 +84,82 @@ def load_et_products(self):
+        for release in releases:
+            attributes = release["attributes"]
+            name = attributes.pop("name")
+            relationships = release.pop("relationships")
+            release, created = CollectorErrataRelease.objects.update_or_create(
+                et_id=release.pop("id"),
+                name=name,
+                defaults={
+                    "is_active": attributes.pop("is_active"),
+                    "enabled": attributes.pop("enabled"),
+                    "brew_tags": relationships.pop("brew_tags"),
+                    "meta_attr": attributes,
+                },
+            )
+            if created:
+                logger.info(f"Created ET Release: {name}")
+
+            product_version_ids = []
+            for product_version in relationships["product_versions"]:
+                product_version_ids.append(product_version["id"])
+            product_versions = CollectorErrataProductVersion.objects.filter(
+                et_id__in=product_version_ids
+            )
+            release.product_versions.set(product_versions)
+
+    def load_variants(self) -> None:
         variants = self.get_paged("api/v1/variants", page_data_attr="data")
         for variant in variants:
-            product_version_id = variant["attributes"]["relationships"]["product_version"]["id"]
+            attributes = variant["attributes"]
+            product_version = attributes["relationships"].pop("product_version")
+            product_version_id = product_version["id"]
             try:
                 et_product_version = CollectorErrataProductVersion.objects.get(
                     et_id=product_version_id
                 )
             except CollectorErrataProductVersion.DoesNotExist:
-                logger.warning("Did not find product version with id %s", product_version_id)
+                logger.warning(f"Did not find product version with id {product_version_id}")
```

Should this raise an error instead of just logging a warning?

---

There are many of these warning logs for each run of the load_et_productions task today. Hopefully there will be less once we delete/recreate the models. If that's the case raising an error would make more sense.